src/OpenTelemetry.Api/Trace/Tracer.cs

```diff
@@ -54,78 +54,70 @@ public TelemetrySpan CurrentSpan
         }
 
         /// <summary>
-        /// Starts span.
+        /// Starts root span.
         /// </summary>
         /// <param name="name">Span name.</param>
         /// <param name="kind">Kind.</param>
+        /// <param name="attributes">Initial attributes for the span.</param>
+        /// <param name="links"> <see cref="Link"/> for the span.</param>
+        /// <param name="startTime"> Start time for the span.</param>
         /// <returns>Span instance.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public TelemetrySpan StartSpan(string name, SpanKind kind = SpanKind.Internal)
+        public TelemetrySpan StartRootSpan(string name, SpanKind kind = SpanKind.Internal, IEnumerable<KeyValuePair<string, object>> attributes = null, IEnumerable<Link> links = null, DateTimeOffset startTime = default)
         {
-            // TODO: Open Question - should we have both StartSpan and StartActiveSpan?
-            // Or should we call this method StartActiveSpan?
-            // This method StartSpan is starting a Span and making it Active.
-            // OTel spec calls for StartSpan, and StartActiveSpan being separate.
-            // Need to see if it makes sense for .NET to strictly adhere to it.
-            // Some discussions in Spec: https://github.com/open-telemetry/opentelemetry-specification/pull/485
-            if (!this.ActivitySource.HasListeners())
-            {
-                return TelemetrySpan.NoopInstance;
-            }
-
-            var activityKind = this.ConvertToActivityKind(kind);
-            var activity = this.ActivitySource.StartActivity(name, activityKind);
-            if (activity == null)
-            {
-                return TelemetrySpan.NoopInstance;
-            }
-
-            return new TelemetrySpan(activity);
+            return this.StartSpanHelper(false, name, kind, default, attributes, links, startTime);
         }
 
         /// <summary>
         /// Starts span.
         /// </summary>
         /// <param name="name">Span name.</param>
         /// <param name="kind">Kind.</param>
-        /// <param name="parent">Parent for new span.</param>
+        /// <param name="parentSpan">Parent for new span.</param>
+        /// <param name="attributes">Initial attributes for the span.</param>
+        /// <param name="links"> <see cref="Link"/> for the span.</param>
+        /// <param name="startTime"> Start time for the span.</param>
         /// <returns>Span instance.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Lets modify the summary to say this will not set the started span as the active one.

Reply:
As per the spec and other languages implementation, starting a span does not make it active, should we call it out explicitly here?

Reply:
Yes. This is important for .NET, as the .NET Activity API's StartActivity is making it active (this is violation of spec). The equivalent SHIM api should be StartActiveSpan.

Reply:
Fixed.